src/lib.rs

```diff
@@ -474,7 +344,8 @@ impl fmt::Display for Display<'_> {
 
 // Adapted from the unstable API:
 // https://doc.rust-lang.org/std/primitive.usize.html#method.div_ceil
-/// Division with upward rouding
+// Can be removed on MSRV 1.70.
+/// Division with upward rounding
```

Comment:
```suggestion
// Division with upward rounding
```

for consistency

Reply:
The idea there was that the first few comments are internal, while the last one is the docstring. But it's not a public function, so I can make it consistent if you want.